tests/libwallet_api_tests/main.cpp

```diff
@@ -210,6 +212,69 @@ TEST_F(WalletManagerTest, WalletManagerOpensWallet)
     std::cout << "** seed: " << wallet2->seed() << std::endl;
 }
 
+
+void open_wallet(Bitmonero::WalletManager *wmgr, Bitmonero::Wallet **wallet, const std::string &pass, std::mutex *mutex)
+{
+    if (mutex)
+        mutex->lock();
+    LOG_PRINT_L3("opening wallet with password '" << pass << "'  in thread: " << std::this_thread::get_id());
```

Comment:
As pointed out by sirnarwhal on IRC, this REALLY should not happen. Logging passwords to cleartext files is seriously bad security practice.